haiku/_src/initializers_test.py

```diff
@@ -61,5 +62,47 @@ def test_initializers(self, dtype):
       self.assertEqual(generated.shape, shape)
       self.assertEqual(generated.dtype, dtype)
 
+  @test_utils.transform_and_run
+  def test_invalid_variance_scale(self):
+
+    with self.assertRaisesRegex(ValueError, "scale.*must be a positive float"):
+      initializers.VarianceScaling(scale=-1.0)
+
+    with self.assertRaisesRegex(ValueError, "Invalid `mode` argument*"):
+      initializers.VarianceScaling(mode="foo")
+
+    with self.assertRaisesRegex(ValueError, "Invalid `distribution` argument*"):
+      initializers.VarianceScaling(distribution="bar")
+
+  @test_utils.transform_and_run
+  def test_compute_fans(self):
+    fan_in_out1 = initializers._compute_fans([])
+    self.assertEqual(fan_in_out1, (1, 1))
+    fan_in_out2 = initializers._compute_fans([2])
+    self.assertEqual(fan_in_out2, (2, 2))
+    fan_in_out3 = initializers._compute_fans([3, 4])
+    self.assertEqual(fan_in_out3, (3, 4))
+    fan_in_out4 = initializers._compute_fans([1, 2, 3, 4])
+    self.assertEqual(fan_in_out4, (6, 8))
+
+  @parameterized.parameters(np.float32, jnp.float32)
+  @test_utils.transform_and_run
+  def test_orthogonal_invalid_shape(self, dtype):
+    init = initializers.Orthogonal()
+    shape = (20, )
```

Comment:
nit: no trailing space after comma in one-tuples

Reply:
Done! Thanks!